src/directives/formly-field.js

```diff
@@ -20,6 +20,7 @@ function formlyField($http, $q, $compile, $templateCache, $interpolate, formlyCo
     scope: {
       options: '=',
       model: '=',
+      originalModel: '=',
```

Comment:
Could you make this an optional attribute by changing `=` to `=?` please? And then in the controller (right in [here](https://github.com/davincho/angular-formly/blob/master/src/directives/formly-field.js#L87-L92)), could you add a default for that value? Something like:

```javascript
formlyUtil.reverseDeepMerge(options, {
  originalModel: options.model,
  extras: {},
  data: {},
  templateOptions: {},
  validation: {}
});
```

So if the `original-model` isn't specified (in the case of someone using the directive directly), it defaults to the given model.

And then we'd need a test to verify this defaulting as well. Thanks!